OpenOversight/tests/routes/test_singular_redirects.py

```diff
@@ -20,29 +20,58 @@
 from OpenOversight.tests.routes.route_helpers import login_admin, login_user
 
 
-def test_redirect_get_started_labeling(client, session):
+@pytest.mark.parametrize(
+    "route",
+    [
+        ("main.redirect_get_started_labeling", "main.get_started_labeling"),
+        ("main.redirect_add_officer", "main.add_officer"),
+        ("main.redirect_add_unit", "main.add_unit"),
+    ],
+)
+def test_redirect_no_params(client, session, route):
     with current_app.test_request_context():
-        resp_no_redirect = client.get(
-            url_for("main.redirect_get_started_labeling"), follow_redirects=False
-        )
+        login_admin(client)
+        resp_no_redirect = client.get(url_for(route[0]), follow_redirects=False)
         with client.session_transaction() as session:
             flash_message = dict(session["_flashes"]).get("message")
 
         assert resp_no_redirect.status_code == HTTPStatus.PERMANENT_REDIRECT
         assert flash_message == FLASH_MSG_PERMANENT_REDIRECT
 
-        resp_redirect = client.get(
-            url_for("main.redirect_get_started_labeling"), follow_redirects=True
-        )
+        resp_redirect = client.get(url_for(route[0]), follow_redirects=True)
         assert resp_redirect.status_code == HTTPStatus.OK
-        assert resp_redirect.request.path == url_for("main.get_started_labeling")
+        assert resp_redirect.request.path == url_for(route[1])
 
 
-def test_redirect_sort_images(client, session):
+@pytest.mark.parametrize(
+    "route",
+    [
+        ("main.redirect_sort_images", "main.sort_images"),
+        ("main.redirect_edit_department", "main.edit_department"),
```

Comment:
Could we use 2 names (1 for each value) here? Like `"source_route, target_route"`

Reply:
Done!